cli/src/merge_tools/mod.rs

```diff
@@ -185,10 +185,7 @@ fn editor_args_from_settings(
 
 /// Resolves builtin merge tool name or loads external tool options from
 /// `[merge-tools.<name>]`.
-pub fn get_tool_config(
-    settings: &UserSettings,
-    name: &str,
-) -> Result<Option<MergeTool>, ConfigError> {
+fn get_tool_config(settings: &UserSettings, name: &str) -> Result<Option<MergeTool>, ConfigError> {
```

Comment:
nit: should be in parent commit?

Reply:
It's `s/pub//` that caused reformatting, and `get_tool_config()` was still referenced by diff_util in the previous commit.

Reply:
Ah, I see. I tried to see what changed but I didn't spot that difference.